dapp/views/smartlocker/smartlocker.jsx

```diff
@@ -20,7 +22,7 @@ const SmartLocker = ({ locker }) => {
     case 'smart':
       return <ManagementSubview/>
     default:
-      return <Bouncing/>
+      return <Bouncing class="bouncing"/>
```

Comment:
`class` is not valid react parameter (`className` is). Why was this added? Bouncing already has its own class name.

Reply:
OK, I see it corresponds with `.bouncing` class above. Changing styles of reusable, external components is frowned upon. I would rather wrap it in div:

```
<div className="loading-subview">
  <Bouncing>
</div>
```
and change `.bouncing` to `.loading-subview` accordingly. 

This place still needs some polish. Will do after demo.

Reply:
Done